internal/sink/matcher.go

```diff
@@ -30,26 +26,22 @@ func (c *ClientMock) Do(req *http.Request) (*http.Response, error) {
 // move to subpackage and change to Interface
 type Matcher interface {
 	IsMatch(*wrp.Message) bool
-
-	//TODO: not sure if this will be functionality of all webhooks or just v1
-	//leaving for now - will make changes if running into roadblock with this
-	getUrls() *ring.Ring
 }
 
 type MatcherV1 struct {
 	events  []*regexp.Regexp
 	matcher []*regexp.Regexp
-	urls    *ring.Ring
```

Comment:
moved urls to sink.go (webhookV1)

Reply:
any functionality related to urls that was in matcher.go has also been moved to sink.go